content/topics/node-and-sql/_index.md

```diff
@@ -2,4 +2,16 @@
 _db_id: 44
 content_type: topic
 title: Introduction to Node and SQL
----
\ No newline at end of file
+---
+
+#### In this topic you will learn how to connect to a postgreSQL database using node
+- node-postgres is a collection of Node.js modules used to interact with a PostgreSQL database.
+- node-postgres has many features like callbacks, connection pooling, prepared statements, async/await and promises.
```

Comment:
maybe you can include a link on callbacks. I found an easy summary: https://www.freecodecamp.org/news/javascript-callback-functions-what-are-callbacks-in-js-and-how-to-use-them/ and async/await and promises so they have an idea as well. Connection pooling is already covered below so you don't have to include a link. 
What do you think?